private-templates-service/adaptivecards-templating-client/javascript/.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: node_js
+node_js:
+  - "6"
+  - "6.1"
+  - "5"
+  - "5.11"
```

Comment:
why these 4 versions specifically? also what is this

Reply:
This is the default versions generated by swagger-codegen - all files under `adaptivecards-templating-client/javascript` were generated